README.md

```diff
@@ -5,80 +5,120 @@
   holidays: ["01/01/2014", "03/01/2014"]    # array items are either parseable date strings, or real Date objects
+  extra_working_dates: [nil], # Makes the calendar to consider a weekend day as a working day.
 )
 ```
 
-`extra_working_dates` key makes the calendar to consider a weekend day as a working day.
+### Use a calendar file
 
-A few calendar configs are bundled with the gem (see [lib/business/data]((lib/business/data)) for
-details). Load them by calling the `load` class method on `Calendar`. The
-`load_cached` variant of this method caches the calendars by name after loading
-them, to avoid reading and parsing the config file multiple times.
+Defining a calendar as a Ruby object may not be convenient, so we provide a way of defining these calendars as YAML. Below we will walk through the necessary [steps](#example) to build your first calendar. All keys are optional and will default to the following:
 
-```ruby
-calendar = Business::Calendar.load("weekdays")
-calendar = Business::Calendar.load_cached("weekdays")
-```
+- If `working_days` is missing, then common default is used (mon-fri).
+- If `holidays` is missing, "no holidays" assumed.
+- If `extra_working_dates` is missing, then no changes in `working_days` will happen.
 
-If `working_days` is missing, then common default is used (mon-fri).
-If `holidays` is missing, "no holidays" assumed.
-If `extra_working_dates` is missing, then no changes in `working_days` will happen.
+> Note: Elements of `holidays` and `extra_working_dates` may be either strings that `Date.parse()` can understand, or YYYY-MM-DD (which is considered as a Date by Ruby YAML itself).
 
-Elements of `holidays` and `extra_working_dates` may be
-eiter strings that `Date.parse()` can understand,
-or YYYY-MM-DD (which is considered as a Date by Ruby YAML itself).
+#### Example
 
 ```yaml
+# lib/calendars/my_calendar.yml
+working_days:
+  - Monday
+  - Wednesday
+  - Friday
 holidays:
-  - 2017-01-08  # Same as January 8th, 2017
+  - 1st April 2020
+  - 2021-04-01
+extra_working_dates:
+  - 9th March 2020 # A Saturday
 ```
 
-### Checking for business days
+Ensure the calendar file is saved to a directory that will hold all your calendars, eg; `lib/calendars` then add this directory to `load_paths` before you call your calendar. `load_paths` can also accept an array of hashes, keyed with the calendar's name.
+
+```ruby
+Business::Calendar.load_paths = [
+  "lib/calendars",
+  { "foo_calendar" => { "working_days" => ["monday"] } },
+]
+```
 
-To check whether a given date is a business day (falls on one of the specified
-working days or working dates, and is not a holiday), use the `business_day?`
-method on `Calendar`.
+Now you can load the calendar by calling the `Business::Calendar.load(calendar_name)`. The
+`Business::Calendar.load_cached(calendar_name)` variant of this method caches the calendars by name after loading them, to avoid reading and parsing the config file multiple times.
```

Comment:
```suggestion
Now you can load the calendar by calling the `Business::Calendar.load(calendar_name)`. 

In order to avoid parsing the calendar file multiple times, there is a `Business::Calendar.load_cached(calendar_name)` method that caches the calendars by name after loading them.
```